src/screens/WalletScreen.js

```diff
@@ -163,16 +162,13 @@ export default function WalletScreen() {
     [network]
   );
 
-  const isCoinListEditedValue = useCoinListEditedValue();
-
   const isLoadingAssets = useSelector(state => state.data.isLoadingAssets);
 
   return (
     <WalletPage testID="wallet-screen">
       {ios && <StatusBar barStyle="dark-content" />}
       {/* Line below appears to be needed for having scrollViewTracker persistent while
       reattaching of react subviews */}
-      <Animated.View style={{ opacity: isCoinListEditedValue }} />
```

Comment:
Why was this line removed? Is the preceding comment no longer true?

Reply:
Yes. This was an issue wiht reanimated v1, removed the comment

Reply:
actually, the comment was referring to the line below, this was a mistake that we put this line here (between the comment and the next line). So now the component and comments are in good positions